src/header/header.scss

```
@@ -292,54 +291,104 @@
         }
       }
     }
+  }
+
+  .iui-button-split-menu {
+    height: 100%;
+    padding: 0 4px 0 12px;
+
+    svg {
+      fill: t(iui-icons-color-actionable);
+    }
+
+    &:hover {
+      background-color: rgba(
+        t(iui-color-foreground-body-rgb),
+        t(iui-opacity-6)
+      );
+      border-color: transparent;
+    }
+
+    .iui-button {
+      padding: 0;
+
+      &:hover {
+        background-color: transparent;
+      }
+    }
 
     .iui-split-menu {
-      display: inline-flex;
-      justify-content: center;
-      @include iui-focus;
+      padding: $iui-xxs;
 
       svg {
-        width: 16px;
-        @include themed {
-          fill: var(--iui-icons-color-actionable);
-        }
+        border-radius: 50%;
       }
 
-      &.iui-active {
+      &:hover {
         svg {
-          fill: var(--iui-icons-color-primary);
+          background-color: rgba(
+            t(iui-color-foreground-body-rgb),
+            t(iui-opacity-6)
+          );
+          border-color: transparent;
         }
       }
+    }
 
-      svg {
-        border-radius: 50%;
-        padding: $iui-xxs;
+    &.iui-active {
+      .iui-label,
+      .iui-icon {
+        color: t(iui-color-foreground-primary);
+        fill: t(iui-icons-color-primary);
       }
 
-      &:hover,
-      &:focus-visible {
-        svg {
+      
+      
+      &,
+      &:hover {
+        background-color: rgba(
+          t(iui-color-foreground-primary-rgb),
+          t(iui-opacity-6)
+        );
+
+        svg.iui-header-button-icon {
           @include themed {
             background-color: rgba(
-              var(--iui-color-foreground-body-rgb),
-              var(--iui-opacity-5)
+              t(iui-color-foreground-primary-rgb),
+              t(iui-opacity-6)
             );
           }
         }
       }
 
-      &.iui-active {
-        &:hover {
-          svg {
-            @include themed {
-              background-color: rgba(
-                t(iui-color-foreground-primary-rgb),
-                t(iui-opacity-5)
-              );
-            }
+      &:focus,
+      &:focus-visible {
+        @include themed {
+          box-shadow: t(iui-focus-box-shadow),
+            0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+        }
+      }
+
+      &,
+      &:focus:not(:focus-visible) {
+        @include themed {
+          box-shadow: 0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+        }
+      }
```

Did you mean to do this?

```suggestion
      &:focus {
        @include themed {
          box-shadow: t(iui-focus-box-shadow),
            0 $iui-xxs 0 0 t(iui-color-foreground-primary);
        }
      }

      &:focus:not(:focus-visible) {
        @include themed {
          box-shadow: 0 $iui-xxs 0 0 t(iui-color-foreground-primary);
        }
      }
```

Because that first selector (`&:focus, &:focus-visible`) will break all focus styling in Safari/IE.

And the second selector (`&`) also applies those styles when focus is not applied, which is wrong. If unsure which selectors to use, you can look at the compiled css to see what the `&` is doing.